core-services/prow/02_config/_plugins.yaml

```diff
@@ -1986,15 +1986,6 @@ external_plugins:
     - pull_request
     name: needs-rebase
   containers/buildah:
-  - endpoint: http://refresh
```

Comment:
why did you remove these?

Reply:
we dont use them in other containers/ repos.  I figured I would remove cruft.  Would you advice differ?